docs/setup-robusta/installation/index.rst

```diff
@@ -16,10 +14,33 @@ Installation Guides
    standalone-installation
    dev-setup
 
-Robusta can be installed three ways:
 
-* :ref:`install-all-in-one` *(recommended, includes Robusta + Prometheus)*
-* :ref:`install-existing-prometheus` *(or a Prometheus compatible metrics-store)*
-* :ref:`install-barebones` *(no Prometheus at all)*
+.. grid:: 1 1 2 2
+    :gutter: 2
+
+    .. grid-item-card:: :octicon:`book;1em;` Monitor Kubernetes from Scratch
+        :class-card: sd-bg-text-light
+        :link: all-in-one-installation
+        :link-type: doc
+
+        Five minute setup. Great default alerts. Powered by Prometheus and Robusta.
+
+    .. grid-item-card::  :octicon:`book;1em;`  Integrate with Existing Prometheus
+        :class-card: sd-bg-light sd-bg-text-light
+        :link: extend-prometheus-installation
+        :link-type: doc
+
+        Make your existing alerts better. Attach pod logs. Automatic alert insights.
+
+Don't want Prometheus? Use :ref:`Robusta without Prometheus <install-barebones>`.
+
+
+Learn Robusta
+^^^^^^^^^^^^^^^^
+
```

Comment:
Can we add some context before the links? (On a page called "Get Started" it made sense to include tutorials. On a page called "Installation" we need to explain why they are there.)

I suggest:

"Already installed Robusta? See what you can do with it."

Reply:
Can we update the list of tutorials as follows? (Remove the original links and of course fix the links I did to be proper sphinx links.)

* [Route alerts to different teams based on namespace, alertname, and more](https://docs.robusta.dev/remove-getstarted/tutorials/index.html#notification-routing)
* [Enhance Prometheus alerts with Robusta](https://docs.robusta.dev/remove-getstarted/tutorials/alert-builtin-enrichment.html)
* [Define new Prometheus alerts](https://docs.robusta.dev/remove-getstarted/tutorials/alert-custom-prometheus.html)
* [Configure auto-remediate for Prometheus alerts](https://docs.robusta.dev/remove-getstarted/tutorials/alert-remediation.html)
* [Track Kubernetes errors and changes using simple YAML](https://docs.robusta.dev/remove-getstarted/tutorials/index.html#custom-alerts-and-playbooks)